Tella/Scenes/Settings/Views/Servers/Uwazi/Step3/UwaziLoginView.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  UwaziLoginView.swift
+//  Tella
+//
+//  Created by Robert Shrestha on 4/24/23.
+//  Copyright © 2023 INTERNEWS. All rights reserved.
+//
+
+import SwiftUI
+
+struct UwaziLoginView: View {
+    @EnvironmentObject var serverViewModel : UwaziServerViewModel
+    @EnvironmentObject var serversViewModel : ServersViewModel
+
+    @Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
+    @EnvironmentObject var mainAppModel : MainAppModel
+
+    @State var presentingSuccessLoginView : Bool = false
+
+    var body: some View {
```

Comment:
After modification the body view has became under some views, I think it's better to move it above these views

Reply:
Done